### ISSUE TITLE
Wrong ifttt_key length

### ISSUE THREAD

**Issue by bwidawsk** [fireflyelectronix/wifidoorbell#9]
The key for IFTTT (for my webhook) is 43 characters. 

> lhp_zDAd2Q6wy9hGW5xd2gAmfXB5ZNH6t01shBii6zs

Something like this should work, and it shouldn't hurt older, shorter keys.

```
diff --git a/WiFi-Doorbell-DB100/src/ConfigPortal.h b/WiFi-Doorbell-DB100/src/ConfigPortal.h
index b35bcb1..1f9a2df 100644
--- a/WiFi-Doorbell-DB100/src/ConfigPortal.h
+++ b/WiFi-Doorbell-DB100/src/ConfigPortal.h
@@ -9,7 +9,7 @@ char mqtt_port[6] = "1883"; //default mqtt port
 char mqtt_username[20];
 char mqtt_password[20];
 char mqtt_topic[100] = "doorbellsensor/status"; //default mqtt topic
-char ifttt_key[40];
+char ifttt_key[44];

 //flag for saving data
 bool shouldSaveConfig = false;
@@ -74,7 +74,7 @@ void configPortal (){
   WiFiManagerParameter custom_mqtt_username("username", "mqtt username", mqtt_username, 20);
   WiFiManagerParameter custom_mqtt_password("password", "mqtt password", mqtt_password, 20);
   WiFiManagerParameter custom_mqtt_topic("topic", "mqtt topic", mqtt_topic, 100);
-  WiFiManagerParameter custom_ifttt_key("ifttt", "ifttt key", ifttt_key, 40);
+  WiFiManagerParameter custom_ifttt_key("ifttt", "ifttt key", ifttt_key, 44);

   //WiFiManager
   //Local intialization. Once its business is done, there is no need to keep it around`

```
Note I bricked my device with this patch when updating via OTA... so bummer.

**Comment by bwidawsk** [comment]
Correction: The problem seems to be that with firmware I build, I get an IP of 192.168.4.1, instead of the documented 192.168.244.1. Sounds like this is a known issue with flash data corruption that is fixed in the latest esp core.

The other option would be to document exactly which board version to use, which isn't there.

**Comment by fireflyelectronix** [comment]
This is fixed now in version 1.3
